datalake/src/main/java/com/sequenceiq/datalake/flow/dr/validation/DatalakeBackupValidationFailureReason.java

```diff
@@ -0,0 +1,16 @@
+package com.sequenceiq.datalake.flow.dr.validation;
+
+public enum DatalakeBackupValidationFailureReason {
+    BACKUP_VALIDATION_ON_UPGRADE("Backup validation on upgrade"),
```

Comment:
Should we use the original reason here instead of duplicating?

Reply:
Ah I see, I didn't think about that. Kalyan added it originally. I am okay with reusing that, I'll make those changes.